src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/RestClients/ResourceManagerRestClientBase.cs

```diff
@@ -219,27 +220,39 @@ public ResourceManagerRestClientBase(Components.HttpClientHelper httpClientHelpe
         {
             using (var httpClient = this.httpClientHelper.CreateHttpClient())
             {
-                var response = await httpClient
-                    .SendAsync(request: request, cancellationToken: cancellationToken)
-                    .ConfigureAwait(continueOnCapturedContext: false);
-
-                if (!response.StatusCode.IsSuccessfulRequest())
+                try
                 {
-                    var errorResponse = await ResourceManagerRestClientBase
-                        .TryReadErrorResponseMessage(response, rewindContentStream: true)
+                    var response = await httpClient
+                        .SendAsync(request: request, cancellationToken: cancellationToken)
```

Comment:
Checked server code. Our practice seems to be putting them at separate lines. Will follow the practice here.